lib/typescript/render/providers/facebook/components/QuickReplies/index.tsx

```diff
@@ -1,12 +1,14 @@
 import React from 'react';
-import styles from './index.module.scss';
+
 import {DefaultRenderingProps} from '../../../../components/index';
 import {Text} from '../../../../components/Text';
 import {Video} from '../../../../components/Video';
 import {Image} from '../../../../components/Image';
 import {QuickReply, AttachmentUnion} from '../../facebookModel';
 import {ImageWithFallback} from 'render/components/ImageWithFallback';
 
+import styles from './index.module.scss';
+
 export type QuickRepliesRenderProps = DefaultRenderingProps & {
```

Comment:
As I understand it:

- DefaultRenderingProps only contains the prop `fromContact`
- It's in the componets dir (but it would make more sense to declare it in lib/render/props.ts?)
- Whenever we use fromContact we actually have the message as a prop  (and if we don't, I'd add that)

Given this situation I would suggest we remove the default props all together. Alternatively, we could move it to the props.ts file and have the message as a default (which is not a bad default no?). Both approaches seem valid to me